src/main/java/nl/tudelft/dnainator/ui/widgets/FileOpenPane.java

```diff
@@ -0,0 +1,29 @@
+package nl.tudelft.dnainator.ui.widgets;
+
+import javafx.scene.layout.GridPane;
+import nl.tudelft.dnainator.ui.widgets.animations.SlidingAnimation;
+
+/**
+ * This class realises a pane from which users can open files. It can be toggled to show/hide.
+ */
+public class FileOpenPane extends GridPane {
```

Comment:
I don't think having this class is necessary. You can just as well create and control the animation from the controller.